components/script/script_thread.rs

```diff
@@ -570,6 +574,14 @@ impl ScriptThreadFactory for ScriptThread {
 }
 
 impl ScriptThread {
+    pub fn set_mutation_observer(observer: &MutationObserver) {
```

Comment:
The "better yet" solution can't work; static methods can't be imported like that.